app/src/main/res/layout/widechat_search_layout.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:fresco="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_alignParentEnd="true"
+    android:orientation="horizontal"
+    android:paddingEnd="8dp"
+    android:paddingStart="0dp">
+
+    <com.facebook.drawee.view.SimpleDraweeView
+        android:id="@+id/profile_image_avatar"
+        android:layout_width="44dp"
+        android:layout_height="44dp"
+        android:layout_marginRight="24dp"
```

Comment:
For future consideration android studio says this about layout_marginRight:
"Consider replacing android:layout_marginRight with android:layout_marginEnd="24dp" to better support right-to-left layouts  Using Gravity#LEFT and Gravity#RIGHT can lead to problems when a layout is rendered in locales where text flows from right to left. Use Gravity#START and Gravity#END instead. Similarly, in XML gravity and layout_gravity attributes, use start rather than left."